crac-plugin/src/test/groovy/io/micronaut/gradle/crac/BaseCracGradleBuildSpec.groovy

```diff
@@ -40,8 +40,7 @@ abstract class BaseCracGradleBuildSpec extends AbstractGradleBuildSpec {
     String getPluginsBlock() {
         """
             plugins {
-                id "io.micronaut.minimal.application"
-                id "io.micronaut.docker"
+                id "io.micronaut.application"
```

Comment:
I would keep the test using `minimal.application` and `docker`, possibly shuffling order so that you check that it works when `crac` comes first.

Reply:
Good idea!  Pushed https://github.com/micronaut-projects/micronaut-gradle-plugin/pull/524/commits/a1eb3bc720680375b2ae0098ddcfcd24b79e7c6b